mailerlite.com.website.json

```diff
@@ -0,0 +1,33 @@
+{  
+   "providerId": "mailerlite.com",
+   "providerName": "MailerLite",
+   "serviceId": "website",
+   "serviceName": "MailerLite Sites",
+   "version": 1,
+   "description": "Create websites or landing pages using your own domain.",
+   "syncPubKeyDomain": "mailerlite.com",
+   "variableDescription": "cnamehost is the subdomain and ip is the IP address of the server.",
+   "records": [
+      {
+         "groupId": "domain",
+         "type": "A",
+         "host": "@",
+         "pointsTo": "%ip%",
+         "ttl": 3600
+      },
+      {
+         "groupId": "redirect",
+         "type": "CNAME",
+         "host": "www",
+         "pointsTo": "pages.mailerlite.com",
+         "ttl": 3600
+      },
+      {
+         "groupId": "subdomain",
+         "type": "CNAME",
+         "host": "%cnamehost%",
+         "pointsTo": "pages.mailerlite.com",
```

Comment:
can we not pass these as variables too ? like we do for IP it make sense to add varibales to everything (just incase we chnage them later on)

Reply:
This was updated as variable as requested.